src/ray/raylet/local_object_manager.h

```diff
@@ -60,9 +58,7 @@ class LocalObjectManager {
         min_spilling_size_(min_spilling_size),
         num_active_workers_(0),
         max_active_workers_(max_io_workers),
-        is_plasma_object_spillable_(is_plasma_object_spillable),
-        restore_object_from_remote_node_(restore_object_from_remote_node),
-        is_external_storage_type_fs_(is_external_storage_type_fs) {}
```

Comment:
Don't we still need `is_external_storage_type_fs_` to determine whether we provide a non-nil spilled node ID when reporting the spilled URL? Right now, it looks like it's not getting properly set.

Reply:
Oh, great catch. Let me also try to add a test for this.